app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java

```diff
@@ -156,7 +156,7 @@ public void showRepositoryError() {
 
     public void showSubscriberDetails(User userInfo) {
         Picasso.with(getContext()).load(userInfo.getAvatarUrl()).into(profileImage);
-        profileFullName.setText(Utils.setUnderlineText(userInfo.getName()));
+        profileFullName.setText(userInfo.getName() != null ? Utils.setUnderlineText(userInfo.getName()) : userInfo.getName());
```

Comment:
This should be `userInfo.getName() != null ? Utils.setUnderlineText(userInfo.getName()) : ""`